roles/postgresql-databases/tasks/main.yml

```diff
@@ -2,18 +2,19 @@
 
 - name: Make sure the PostgreSQL databases are present
   become: true
-  become_user: postgres
+  become_user: "{{ patroni_superuser_username }}"
```

Comment:
patroni_superuser_username defines the database user, while in become_user we must specify the system user.

patroni_superuser_username may differ from the system user name and there may not be such a system user at all and we will get an error.